docker/Dockerfile

```diff
@@ -36,7 +36,7 @@ RUN cmake \
     ../src
 RUN make VERBOSE=1 -j2
 RUN ccache -s
-RUN ctest -E "(walker:SkewNormalPDF_pe8|walker:DiagOrnsteinUhlenbeckPDF_pe8)"
```

Comment:
I would do `Release` build instead and not exclude these. See https://github.com/quinoacomputing/quinoa/pull/108#issuecomment-283411905.